tests/handlers/tests.py

```diff
@@ -38,7 +38,8 @@ def test_bad_path_info(self):
         environ['PATH_INFO'] = b'\xed' if six.PY2 else '\xed'
         handler = WSGIHandler()
         response = handler(environ, lambda *a, **k: None)
-        self.assertEqual(response.status_code, 400)
+        # The path of the request will be encoded to '/%ED'.
```

Comment:
OK I spotted the `get_script_name()` call inside the `WSGIRequest` constructor, https://github.com/django/django/blob/master/django/core/handlers/wsgi.py#L81 but it seems we blow up due to an earlier call in the handler. I'm happy with removing the try/except block.